patches/chromium/osr_shared_texture_remove_keyed_mutex_on_win_dxgi.patch

```diff
@@ -0,0 +1,85 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Reito <carolwolfking@gmail.com>
+Date: Fri, 19 Apr 2024 11:46:47 +0800
+Subject: Remove DXGI GMB keyed-mutex
+
+This CL is only for reference for a vendor specific change. It is kept for rebasing.
+
+On CEF/electron, FrameSinkVideoCapturer is used in GPU accelerated OSR, and it can be faster
+if we remove the mutex on the shared resource for Windows.
+
+Maintain this patch in https://crrev.com/c/5465148
+
+Change-Id: Id90acd587f17acd228ae7cb5ef93005eb8388b80
```

Comment:
Is this patch required for this feature? Why is this mutex there in the first place? Why can't this CL be merged upstream?

Reply:
It is not mandatory, but can provide an extra performance gain. Also as commented by OBS teams, texture mutex have performance spikes when acquiring them, and it may cause driver failure in extreme condition or old systems. Also according to the use case by OSR, the texture will not be written to again so a mutex is not requried. It is a tiny patch and I think it can be maintained easily, in general I think is worth to patch to eliminate performance spikes.
It is not possible to merge into upstream, as OSR is not the only use case in chromium, it was used as GMB (GpuMemoryBuffer) so a mutex is needed in some other code paths.

Reply:
Okay, that makes sense. Would you mind expanding the patch description with some of the info from this comment?